src/Panoviewer.js

```diff
@@ -146,18 +148,23 @@ class PanoViewer {
                 hs = new Hotspot(hotspot.id, config.CAMERA_HEIGHT, hotspot.distance, hotspot.yaw, hotspot.year)
                 hs.element.firstChild.addEventListener('click', () => this._updatePanorama(hotspot.id));
                 container.createHotspot(hs.element, hs.position);
-            }
-        );
+            });
         } catch (e) {
+            // `console.error` a more discriptive error message
             console.log(e);
         }
     }
 
     _addCallbacks() {
-        let event_register = null;
         for (let evt of Object.keys(config.CALLBACKS)) {
+            // Camel casing...
+            let event_register = null;
+
             if (evt === 'active' || evt === 'inactive') {
                 event_register = this.viewer.controls();
+            // So 'location' events will not be registered...?
+            // I see that callback will be called in the _updatePanoData
+            // function. Make a notion of that in a comment here?
```

Comment:
location will be triggered by location change, it is not generated as an event. That is why it is not registered. There is notation about tit in the docs